### ISSUE TITLE
Lite Wallet

### ISSUE THREAD

**Issue by lightspeed393** [SpaceWorksCo/roadmap#4]
A lite wallet is important for users who don't wish to download the entire blockchain.

It's important to note that staking will not be available when using the lite wallet.

The goal is to fork [electrum-komodo](https://github.com/komodoplatform/electrum-komodo) and create an electrum-spacecoin version.

We already have electrum servers running at `electrum1.spaceworks.co` and `electrum2.spaceworks.co`.

AtomicDEX-Pro will also function as a lite wallet for SPACE after #3 is complete.

**Comment by lightspeed393** [comment]
I made a [electrum-spacecoin](https://github.com/spaceworksco/electrum-spacecoin) fork of [electrum-komodo](https://github.com/pbca/electrum-komodo) `RICK` branch and started working on changes for SPACE.

**Comment by lightspeed393** [comment]
The first version of electrum-spacecoin (3.3.0) wallet has been released:

[https://github.com/SpaceWorksCo/electrum-spacecoin/releases/tag/spacecoin-3.3.0-beta](https://github.com/SpaceWorksCo/electrum-spacecoin/releases/tag/spacecoin-3.3.0-beta)

**Comment by lightspeed393** [comment]
Moved to done since a lite wallet does now exist. 

I'll keep the issue open until more testings/updates are done to electrum-spacecoin or another issue is opened covering such things.

**Comment by lightspeed393** [comment]
[HyperDEX](https://github.com/spaceworksco/hyperdex) also functions as a lite wallet for SPACE.

**Comment by lightspeed393** [comment]
[AtomicDEX-Desktop](https://github.com/komodoplatform/atomicdex-desktop) also functions as a Lite Wallet.
